src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.ts

```diff
@@ -250,21 +250,25 @@ export class InstructorSessionEditPageComponent extends InstructorSessionBasePag
       modalRef.componentInstance.sessionToCopyCourseId = this.courseId;
 
       modalRef.result.then((result: CopySessionModalResult) => {
-        this.feedbackSessionsService.getFeedbackSession({
-          courseId: this.courseId,
-          feedbackSessionName: this.feedbackSessionName,
-          intent: Intent.FULL_DETAIL,
-        }).pipe(
-            switchMap((feedbackSession: FeedbackSession) =>
-                this.copyFeedbackSession(feedbackSession, result.newFeedbackSessionName, result.copyToCourseId)),
-        ).subscribe((createdSession: FeedbackSession) => {
-          this.navigationService.navigateWithSuccessMessage(this.router,
-              '/web/instructor/sessions/edit',
-              'The feedback session has been copied. Please modify settings/questions as necessary.',
-              { courseid: createdSession.courseId, fsname: createdSession.feedbackSessionName });
-        }, (resp: ErrorMessageOutput) => { this.statusMessageService.showErrorToast(resp.error.message); });
-      }, () => {});
-    }, (resp: ErrorMessageOutput) => { this.statusMessageService.showErrorToast(resp.error.message); });
+        result.copyToCourseList.forEach((copyToCourseId: string) => {
+          this.feedbackSessionsService.getFeedbackSession({
+            courseId: this.courseId,
+            feedbackSessionName: this.feedbackSessionName,
+            intent: Intent.FULL_DETAIL,
+          }).pipe(
+              switchMap((feedbackSession: FeedbackSession) =>
+                  this.copyFeedbackSession(feedbackSession, result.newFeedbackSessionName, copyToCourseId)),
+          ).subscribe((createdSession: FeedbackSession) => {
+            if (result.copyToCourseList.length > 1) { return; }
```

Comment:
If there are multiple course IDs, we still need to show toast mentioning that the operation is successful (and if any are failing).